examples/player/simid_player.js

```diff
@@ -144,8 +162,12 @@ class SimidPlayer {
     // The target of the player to send messages to is the newly
     // created iframe.
     playerDiv.appendChild(simidIframe);
-    // Set up css to overlay the SIMID iframe over the video creative.
-    simidIframe.classList.add('simid_creative');
+    // Set up css to overlay the SIMID iframe over the entire video creative
+    // only if linear.
+    if (this.isLinearAd_){
+      simidIframe.classList.add('simid_creative');
```

Comment:
I think it makes sense to change the name of this css class, if its only for linear ads now to indicate as such

Reply:
Good spot- never changed it from original player